spec/models/issue_spec.rb

```diff
@@ -1,6 +1,10 @@
 require 'rails_helper'
 
 describe Issue, type: :model do
+  before(:each) do
+    allow_any_instance_of(Issue).to receive(:geocode).and_return([1,1])
```

Comment:
globally override geocode method (so it will do nothing)
specs need to work without external services, and best for now is just to disable geocoding for seed and tests